src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.Android.cs

```diff
@@ -312,16 +310,11 @@ private static async Task<IDisposable> SetImageBrushAsBackground(CancellationTok
 			});
 		}
 
-		private static void ExecuteWithNoRelayout(BindableView target, Action<BindableView> action)
+		private void ExecuteWithNoRelayout(Action<BindableView> action)
```

Comment:
I wonder if this would become useless with @carldebilly's measure updates.

Reply:
Yes and no. It's _gating_ the invalidation a little sooner, so there's a slight gain doing that.  I'm more wondering about the cost of the delegate here..